src/adm/filter.rs

```diff
@@ -380,14 +380,10 @@ impl AdmFilter {
     ) -> HandlerResult<Option<Tile>> {
         // Use strict matching for now, eventually, we may want to use backwards expanding domain
         // searches, (.e.g "xyz.example.com" would match "example.com")
-        match self
-            .advertiser_filters
-            .adm_advertisers
-            .get(&tile.name.to_lowercase())
-        {
+        match self.advertiser_filters.adm_advertisers.get(&tile.name) {
```

Comment:
This does mean that "Amazon" and "amazon" are different companies. Do we want that?

Reply:
(I think we want to keep the `name.to_lowercase()` because we're normalizing the names when we add them to `adm_advertisers`.

Reply:
huh i thought i added that back... will re add,

Reply:
Are we not indexing by lowercase? If so, we probably want to remove the lowercase we do in `Deserialize<'de> for AdmAdvertiserSettings`. 

Probably should add a unit test somewhere to ensure that we don't trip on this in the future.

Reply:
<img width="1119" alt="Screen Shot 2022-10-26 at 11 49 04 AM" src="https://user-images.githubusercontent.com/25379936/198111092-35f4292e-f9d1-4f19-a084-51cc3381182f.png">
this is the current diff i see?